### PR TITLE
3.0/feature/iframe

### PR DESCRIPTION
**Pull request by annalinneajohansson**

### CODE REVIEW

**Comment by annalinneajohansson** [comment]
Dependant on https://github.com/helsingborg-stad/styleguide/pull/543 being merged + deployed at the same time

**Comment by sebastianthulin** [comment]
Still in draft, right?

File: assets/source/3.0/sass/embed.scss [hunk 1]
```diff
@@ -22,6 +22,10 @@
 		top: 0;
 		object-fit: cover;
 	}
+
+	.js-suppressed-iframe-wrapper {
+    	position: static;
```
Comment (sebastianthulin) [inline assets/source/3.0/sass/embed.scss:27]: This is default behaviour. Is another position set anywhere else?

Comment (NiclasNorin) [reply assets/source/3.0/sass/embed.scss:27]: This was needed before when the rendering was made from the backend and looked a little different. I dont think this style or special styling of the video module has any affect anymore. I will  remove them!

Comment (NiclasNorin) [reply assets/source/3.0/sass/embed.scss]: Other is removed in the styleguide

**Pull request merged**